src/loaders/gtdb/gtdb_loader_helper.py

```diff
@@ -16,3 +21,33 @@ def convert_to_json(docs, outfile):
 
     with jsonlines.Writer(outfile) as writer:
         writer.write_all(docs)
+
+
+def parse_genome_id(gtdb_accession):
+    """
+    Extract the genome id from the GTDB accession field by removing the first 2 characters.
+
+    e.g. GB_GCA_000016605.1 -> GCA_000016605.1
+         GB_GCA_000172955.1 -> GCA_000172955.1
+    """
+    return gtdb_accession[3:]
+
+
+def init_genome_atrri_doc(kbase_collection, load_version, genome_id):
+    """
+    Create a document with only a '_key' field in ArangoDB for genome attributes collection.
```

Comment:
This makes it sound like it's actually creating the document in ArangoDB rather than just returning a dictionary

Reply:
👍  updated the description.

Reply:
:+1: